pentaho-spring-dm-extender/pom.xml

```diff
@@ -55,7 +56,7 @@
         <extensions>true</extensions>
         <configuration>
           <instructions>
-            <Import-Package>org.springframework.core.io;version="[3.0,4.0)",org.springframework.util;version="[3.0,4.0)",*</Import-Package>
+            <Import-Package>org.springframework.core.io;version="[3.0,5.0)",org.springframework.util;version="[3.0,5.0)",*</Import-Package>
```

Comment:
I'm not sure this makes sense. Spring-DM got deprecated so it probably doesn't exist on spring >= 4.

Reply:
After our investigation on Spring-DM, this doesn't make sense to me neither ... reverting it